<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JavaPage</class>
 <widget class="QWidget" name="JavaPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string notr="true">Tab 1</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="memoryGroupBox">
         <property name="title">
          <string>Memory</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <widget class="QSpinBox" name="maxMemSpinBox">
            <property name="toolTip">
             <string>The maximum amount of memory Minecraft is allowed to use.</string>
            </property>
            <property name="suffix">
             <string notr="true"> MiB</string>
            </property>
            <property name="minimum">
             <number>128</number>
            </property>
            <property name="maximum">
             <number>65536</number>
            </property>
            <property name="singleStep">
             <number>128</number>
            </property>
            <property name="value">
             <number>1024</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelMinMem">
            <property name="text">
             <string>Minimum memory allocation:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelMaxMem">
            <property name="text">
             <string>Maximum memory allocation:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="minMemSpinBox">
            <property name="toolTip">
             <string>The amount of memory Minecraft is started with.</string>
            </property>
            <property name="suffix">
             <string notr="true"> MiB</string>
            </property>
            <property name="minimum">
             <number>128</number>
            </property>
            <property name="maximum">
             <number>65536</number>
            </property>
            <property name="singleStep">
             <number>128</number>
            </property>
            <property name="value">
             <number>256</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelPermGen">
            <property name="text">
             <string notr="true">PermGen:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="permGenSpinBox">
            <property name="toolTip">
             <string>The amount of memory available to store loaded Java classes.</string>
            </property>
            <property name="suffix">
             <string notr="true"> MiB</string>
            </property>
            <property name="minimum">
             <number>64</number>
            </property>
            <property name="maximum">
             <number>999999999</number>
            </property>
            <property name="singleStep">
             <number>8</number>
            </property>
            <property name="value">
             <number>64</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="javaSettingsGroupBox">
         <property name="title">
          <string>Java Runtime</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="labelJavaPath">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Java path:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="javaPathTextBox"/>
            </item>
            <item>
             <widget class="QPushButton" name="javaBrowseBtn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>28</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string notr="true">...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QLineEdit" name="jvmArgsTextBox"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelJVMArgs">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>JVM arguments:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="javaDetectBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Auto-detect...</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="javaTestBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Test</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>minMemSpinBox</tabstop>
  <tabstop>maxMemSpinBox</tabstop>
  <tabstop>permGenSpinBox</tabstop>
  <tabstop>javaBrowseBtn</tabstop>
  <tabstop>javaPathTextBox</tabstop>
  <tabstop>jvmArgsTextBox</tabstop>
  <tabstop>javaDetectBtn</tabstop>
  <tabstop>javaTestBtn</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
